icaruscode/CRT/CRTUtils/CRTPMTMatchingUtils.cxx

```diff
@@ -63,40 +67,67 @@ icarus::crt::CRTMatches icarus::crt::CRTHitmatched(
       .R();
     if (abs(tof) >= interval) continue;
     if (tof < 0) {
-      if (crtHit->plane > 36)
+      if (MatchBottomCRT && crtHit->plane > 49) 
+	bottomen++;
+      else if(crtHit->plane > 49) 
+	continue;
+      else if (crtHit->plane > 36)
         sideen++;
       else
         topen++;
       CRTPMT m_match = {tof, distance, crtHit};
       enteringCRTHits.push_back(std::move(m_match));
     } else if (tof >= 0) {
-      if (crtHit->plane > 36)
+      if (MatchBottomCRT && crtHit->plane > 49)
+	bottomex++;
+      else if(crtHit->plane > 49) 
+	continue;
+      else if (crtHit->plane > 36)
         sideex++;
       else
         topex++;
       CRTPMT m_match = {tof, distance, crtHit};
       exitingCRTHits.push_back(std::move(m_match));
     }
   }
-  if (topen == 0 && sideen == 0 && topex == 0 && sideex == 0)
-    flashType = MatchType::noMatch;
-  else if (topen == 1 && sideen == 0 && topex == 0 && sideex == 0)
-    flashType = MatchType::enTop;
+  if (topen == 0 && sideen == 0 && topex == 0 && sideex == 0){
+    if(MatchBottomCRT){
+      if(bottomex==0 && bottomen==0) flashType = MatchType::noMatch;
+      else if (bottomex>=1 && bottomen==0) flashType = MatchType::exBottom;
+      else if (bottomex==0 && bottomen>=1) flashType = MatchType::enBottom;
+    } 
+    else flashType = MatchType::noMatch; 
```

Comment:
I suppose that if `MatchBottomCRT` is `false` the two bottom counters are `0` anyway:
```suggestion
    if(bottomex==0 && bottomen==0) flashType = MatchType::noMatch;
    else if (bottomex>=1 && bottomen==0) flashType = MatchType::exBottom;
    else if (bottomex==0 && bottomen>=1) flashType = MatchType::enBottom;
```